src/deepsparse/benchmark/benchmark_sweep.py

```diff
@@ -343,7 +349,7 @@ def main(
         if model.startswith("zoo:"):
             model_name = ""
         else:
-            model_name = Path(model).name
+            model_name = Path(model).stem
```

Comment:
why do we use stem here?

Reply:
Yes, it would but otherwise the exported filename would contain two dots for example `benchmark_resnet.onnx_2023010212010212.csv` which looks awkward, if needed I think we could add the engine  in the filename.

Reply:
Where are those file names coming from?